tools/mysql.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
```

Comment:
This script does not relate to the original issue. Can you split this PR and add another PR with a information what this script is for?